python/lesson1/tutorial.md

```diff
@@ -1,135 +1,114 @@
-
-    9 * 4 - 6
-
-Now try a few more. You can combine as many operations as you like.
-
-### Operator Precedence
-
-Did you notice any unexpected results when you started combining operations? If you didn't, try this:
-
-    10 - 2 * 4
-
-Python follows the traditional mathematical rules of precedence, which state that multiplication and division are done before addition and subtraction. (You may remember *BODMAS*.) This means in our example above, 2 and 4 are multiplied first, and then the result is subtracted from 10.
-
-We can change the order of operations by using parentheses. Anything inside parentheses is executed first.
-
-Now try it like this:
-
-    (10 - 2) * 4
-
-You should have a different answer.
-
-Because of precedence rules, complex operations such as our first example can be quite confusing to read. If you find yourself writing more complex expressions, there is no harm in adding parentheses for clarity.
-
-### Decimal Points
-
-Remember dividing 14 by 4? The result probably surprised you. Fortunately, there is a way to convince Python that you really do want the full answer, not just the whole number. We can do this by *starting* with numbers with decimal points:
+## Opening the REPL
 
-    14.0 / 4.0
+Python is often developed with the aid of a *REPL*, or *Read-Eval-Print-Loop*.
+The REPL is a way of getting immediate feedback as you work. Python's REPL is
+called **IDLE**.
 
-Now try a few more and see how it goes. Remember that you can also use non-whole numbers in your calculations.
+It's very important to find what Python version you are working with.
 
-You may notice that there are still some limitations. For example, try:
+On Windows, open `cmd` or on Mac OS / GNU/Linux open `terminal` and type
+(without the `$` sign):
 
-    10.0 / 3.0
+    $ python --version
 
-The answer should go on forever, but it doesn't. This is because there is only a finite amount of space to store the number, and it runs out after approximately 15 significant figures.
+What you should see is:
 
-### Concluding
+    $ python 3.5.2
 
-Now let's combine what we have learnt today. We can tell `print` to print multiple things at once, separated by a comma:
+Any 3.* version should do for this tutorial to work.
 
-    print 'The result of 2 + 2 is', 2 + 2
+Now, open the *REPL* by typing:
 
-This concludes today's tutorial. In the next tutorial, we'll find out how to combine the results of multiple separate expressions using variables, get input from the user, and make decisions based on that information.
+    $ python
 
-### Further Reading
+And you should see something like this:
 
-There is a very good introductory article in [Google Developers Guide](https://developers.google.com/edu/python/introduction)
+    Python 3.5.2 (default, Aug  6 2016, 15:44:26) 
+    [GCC 5.4.0] on linux
+    Type "help", "copyright", "credits" or "license" for more information.
+    >>> 
```

Comment:
Worth explaining how to exit the REPL too, so people can *stop* after the end of the lesson.

(And re-explain how to start it in the next tutorial.)